remote/remote.go

```diff
@@ -207,11 +207,24 @@ func newV1Image(keychain authn.Keychain, repoName string, platform imgutil.Platf
 			}
 			return nil, fmt.Errorf("connect to repo store '%s': %s", repoName, err.Error())
 		}
+		break
```

Comment:
Thanks for catching this. Uggh 300ms extra for every successful image fetch... this might be worthy of a patch WDYT @dfreilich @jabrown85 @jromero ?

Reply:
I'm pro patch if the burden is minimal.

Reply:
Included in on [PR 117](https://github.com/buildpacks/imgutil/pull/117)